### PR TITLE
Docs: Beta-10 Add .env instruction for critical secure

### PR DESCRIPTION
**Pull request by miguelgargallo**
Docs: Beta-10 Add .env instruction for critical secure

### CODE REVIEW

**Comment by miguelgargallo** [comment]
- [x] done

File: app/templates/docs/integration/sdks/nodejs.hbs [hunk 1]
```diff
@@ -41,6 +41,71 @@
...
+	<h3>Additional Configuration for credentials - Optional </h3>
+	
+	When handling credentials in applications,  It is recommended that you add these to a `.env` file 
+	<p> To do so, first install <Link @link="https://www.npmjs.com/package/dotenv">dotenv</Link>  which is an NPM package that loads environment variables from a  <code> .env </code> file</p>
+	<Code @name="docs-integration-libraries-nodejs-extra-install-dotenv.bash">
+		npm install dotenv
+	</Code>
+	<p>Add a <code>.env</code> file to the root of your project. Then input your SurrealDB credentials</p>
+	<Code @name="docs-integration-libraries-nodejs-extra-config.bash">
+		SURREALDB_LINK=http://127.0.0.1:8000/rpc
```
Comment (Ekwuno) [inline app/templates/docs/integration/sdks/nodejs.hbs:54]:
```suggestion
		SURREALDB_URL=http://127.0.0.1:8000/rpc
```

**Comment by miguelgargallo** [comment]
Dear @kearfy why this was approved and was not merged as a Contributor¿?

**Comment by miguelgargallo** [comment]
@kearfy here https://github.com/surrealdb/www.surrealdb.com/pull/200#issuecomment-1623134675

**Comment by kearfy** [comment]
Hi @miguelgargallo, apologies! I have completely missed your PR unfortunately. 

I have taken a look at your suggested change but decided that we will not be merging this. While it absolutely is good practice to maintain connection details and similar in environment variables, it is an opinionated solution. We want to keep the documentation for SurrealDB and it's SDKs as abstract as possible, and don't feel like this is in line with that goal.

**Comment by miguelgargallo** [comment]
It's nothing you are welcome, see you at SurrealDB World 🥰🚀👩‍🚀🌠

**Pull request closed without merging**